addon/serializers/linked-node.js

```diff
@@ -0,0 +1,16 @@
+import OsfSerializer from './osf-serializer';
+
+// TODO Dawn - a linked-node serializer is actually using the node model - Ember doesn't like this.
```

Comment:
I think so - unless you feel this method is suitable.  To add a project (linked-node) to a collection, the request is going to look a little different (mentioned above, needs to look like institutions).  So I created a serializer for the linked-node.  The only problem is, a linked-node is actually a node.  So the model type does not match the serializer.  So I could make a linked-node model, and just extend the node model, but I don't love that.

This technically works, but if you write a test for the linked-node serializer, it will complain because there's no linked-node model.  

Sam said just go with this for now and maybe look into it later.